Hamzah/stocks_challenge.py

```diff
@@ -0,0 +1,91 @@
+print("Challenge 3.2: Playing with the stock market")
+
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+
+Amazon = 3000
```

Comment:
You may not have noticed, but Amazon appears different than the other stocks because you used a capital letter in the variable name. Try to avoid using capital letters or numbers at the beginning of a variable name because that's unconventional. You will learn in the future that some variables can be written in all caps for specific reasons. But for now, try to avoid the caps.